internal/pkg/describe/lb_web_service.go

```diff
@@ -252,6 +265,47 @@ func (e envVars) humanString(w io.Writer) {
 	}
 }
 
+type Secrets struct {
+	Name        string `json:"name"`
+	Environment string `json:"environment"`
+	ValueFrom   string `json:"valueFrom"`
+}
+
+type secrets []*Secrets
+
+func (s secrets) humanString(w io.Writer) {
+	fmt.Fprintf(w, "  %s\t%s\t%s\n", "Name", "Environment", "Value From")
+	fmt.Fprintf(w, "  %s\t%s\t%s\n", "----", "-----------", "----------")
+	var prevName string
+	var prevValueFrom string
+	for _, secret := range s {
+		// If the valueFrom is not an ARN, preface it with "parameter/"
+		var outputValueFrom string
+		_, err := arn.Parse(secret.ValueFrom)
+		if err != nil {
+			outputValueFrom = fmt.Sprintf("parameter/%s", secret.ValueFrom)
+		} else {
+			outputValueFrom = secret.ValueFrom
+		}
```

Comment:
nit: usually whenever there is an else statement, there is an opportunity for refactoring 😄  🧐 
```suggestion
		valueFrom := secret.ValueFrom
		if _, err := arn.Parse(secret.ValueFrom); err != nil {
			// If the valueFrom is not an ARN, preface it with "parameter/"
			valueFrom = fmt.Sprintf("parameter/%s", secret.ValueFrom)
		} 
```

Reply:
Much better, thanks! Done.